tools/vagrant/simple/hello.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+vagrant up
+
+vagrant ssh -c "sudo apt-get install -y git"
+
+vagrant ssh -c "git clone --depth=1 https://github.com/openwhisk/openwhisk.git"
+
+vagrant ssh -c "cd openwhisk/tools/ubuntu-setup && source all.sh"
+
+vagrant reload
+
+vagrant ssh -c "cd openwhisk && tools/db/couchdb/start-couchdb-box.sh whisk_admin some_passw0rd"
```

Comment:
Yep but I don't know what else it's involved in getting any clean happy, I will look into it but for now this is a workaround, I will add a TODO comment